File: app/src/main/java/com/github/wyang/basiclibrarydemo/ShapeActivity.java
package com.github.wyang.basiclibrarydemo;

import android.support.v7.widget.Toolbar;

import butterknife.BindView;

/**
 * Created by fxb on 2020/8/27.
 */
public class ShapeActivity extends BaseActivity {
    @BindView(R.id.toolbar)
    Toolbar mToolBar;

    @Override
    public int getLayoutId() {
        return R.layout.activity_shape_view;
    }

    @Override
    protected void initView() {
        initToolBar(mToolBar, "自定义Shape", true);
    }

}

File: app/src/main/java/com/github/wyang/basiclibrarydemo/ShapeActivity.kt
package com.github.wyang.basiclibrarydemo

import com.github.wyang.basiclibrarydemo.databinding.ActivityShapeViewBinding
import com.github.wyang.basiclibrarydemo.databinding.LayoutToolbarBinding

class ShapeActivity : BaseActivity<ActivityShapeViewBinding>() {
    private var toolbarBinding: LayoutToolbarBinding? = null

    override fun getViewBinding(): ActivityShapeViewBinding {
        val binding = ActivityShapeViewBinding.inflate(layoutInflater)
        toolbarBinding = binding.includeToolbar
        return binding
    }

    override fun initialize() {
        toolbarBinding?.let { initToolBar(it.toolbar, "自定义Shape", true) }
    }
}